docs/cli/index.mdx

```diff
@@ -60,7 +60,7 @@ temporal server start-dev --namespace foo --namespace bar
 Register Namespaces with `namespace create`:
 
 ```shell
-temporal operator namespace create foo
+temporal operator namespace create -n foo
```

Comment:
```suggestion
temporal operator namespace create --namespace foo

temporal operator namespace create -n foo
```

As a tech writing rule, prefer the full "readable" version to the shortcut. Clarity trumps concision. 

Several ways to approach this, use the full version, use both with the preferred first, or stick with organizational standard (-n) and ignore suggestion. This is a due diligence suggestion. 

The extra characters take up more space, especially in help messages, but I was scratching my head a lot when I was first exposed to `-n` in messages and had no context to explain it. As a new adopter being shown --namespace first and then knowing as a Unix user there might be a shorter flag, would have made some work (coverage of creating a namespace using the Temporal CLI) easier for me. I expect other new adopters, a big target I've been told for approachable tooling and documentation, would also benefit.

Reply:
Also, leaving this here because I don't know how to add it to the points where it applies (starting on line 25). The intro can be simplified for easier reading.

Starting at Line 25 -> 


The Temporal CLI, formerly 'tctl', is a command-line interface that offers direct access to a Temporal Cluster (Service).
It supports management, monitoring, and debugging of Temporal Applications, and administrative functions like Namespace management. 
With it, you can initiate, signal, or cancel Workflows and query Workflow states from your terminal.
The Temporal CLI is an indispensable component of your Temporal developer toolkit.

When upgrading from `tctl` to Temporal CLI, update your existing environment variables and be aware that some commands use new syntax.